app/views/simul/form.scala

```diff
@@ -126,6 +128,12 @@ object form {
           help = views.html.tournament.form.positionInputHelp.some
         )(form3.input(_))
       ),
+      form3.group(
+        form("estimatedStartAt"),
+        frag("Estimated start time"),
+        help = frag("Optional estimated start time").some,
```

Comment:
no need to repeat the name of the field in its description, leave it to `none` if there's nothing to add

Reply:
I've interpreted this as I can remove the "help" parameter (seems to default to None),
since it doesn't add any meaningful information...

587afa0